kinto/tests/test_views_buckets.py

```diff
@@ -20,7 +20,8 @@ def setUp(self):
 
     def test_buckets_are_global_to_every_users(self):
         self.app.patch_json(self.record_url,
-                            {'permissions': {'read': [Authenticated]}},
+                            {'data': {},
```

Comment:
Actually with your fix on cliquet if you don't provide data you'll get a 400 (data should be present and is not defined)

See https://github.com/mozilla-services/kinto/issues/63